analysis/report_templates/default.html

```diff
@@ -232,6 +232,27 @@ <h5 class="center-align">Mann-Whitney U test</h4>
                 </li>
             </ul>
 
+            <div class="row">
+                <div class="col s12">
+                    <div class="card white darken-1">
+                        <div class="card-content black-text">
+                        <span class="card-title">Coverage reports</span>
```

Comment:
UI needs to be changed. We need to have a drop down similar to "Sample statistics and statistical significance"
Make it like "Coverage reports" and when someone clicks it, have a list of buttons for fuzzers (no dropdown, why extra click).

Reply:
+1 on having a simple list, instead of a the current dropdown "fuzzer list" button